lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -47,10 +47,14 @@
 // styles together, so that card becomes more of
 // a styled container.
 .beta-card {
-  background-color: $white;
   border-radius: 4px;
   color: $dark-gray;
   overflow: hidden;
+  background-color: $white;
+
+  .beta-card__video {
+    margin-bottom: -10px;
```

Comment:
Can you just add a little `// HACK: xyz` comment above this line explaining why we need this? It's easy to forget otherwise.